src/main/java/edu/ie3/datamodel/utils/validation/ConnectorValidationUtils.java

```diff
@@ -88,15 +131,31 @@ private static void checkLine(LineInput line) {
    * - vRated is greater 0 (Rated voltage)
    *
    * @param lineType Line type to validate
+   * @return a list of try objects either containing an {@link InvalidEntityException} or an empty
+   *     Success
    */
-  protected static void checkLineType(LineTypeInput lineType) {
-    checkNonNull(lineType, "a line type");
-    detectNegativeQuantities(new Quantity<?>[] {lineType.getB(), lineType.getG()}, lineType);
-    detectZeroOrNegativeQuantities(
-        new Quantity<?>[] {
-          lineType.getvRated(), lineType.getiMax(), lineType.getX(), lineType.getR()
-        },
-        lineType);
+  protected static List<Try<Void, InvalidEntityException>> checkLineType(LineTypeInput lineType) {
+    try {
+      checkNonNull(lineType, "a line type");
+    } catch (InvalidEntityException e) {
+      return List.of(
+          new Failure<>(
+              new InvalidEntityException(
+                  "Validation not possible because received object {" + lineType + "} was null",
+                  e)));
+    }
```

Comment:
I think this can be solved more elegantly with the existing `Try.ofVoid()`, right?
Also please check all other changed code for this opportunity to simplify the code (I've found a couple more)